AirportDepartures.playground/Contents.swift

```diff
@@ -75,9 +154,23 @@ import UIKit
 //: d. Call the `alertPassengers()` function on your `DepartureBoard` object below
 //:
 //: f. Stretch: Display a custom message if the `terminal` is `nil`, tell the traveler to see the nearest information desk for more details.
-
-
-
+for flight in departureBoard.flights {
```

Comment:
This loop should ideally go inside of the `alertPasengers()` function in your departure board class. The instructions are confusing on this though, as the step is listed way down here, yet your class is up at the top. I definitely won't hold it against you, but just thought you should know.